README.md

```diff
@@ -94,7 +94,7 @@ Users must select a default network provider. This cannot be changed. Different
 
 The network type is always read from the Cluster configuration.
 
-Currently, the only supported value for network Type is `OpenShiftSDN`.
+Currently, the only supported value for network Type is `OpenShiftSDN`. *All other network types will be ignored.* If you use a third-party network provider that is not managed by the operator, you should set your network type to something meaningful to you.
```

Comment:
Clarify "ignored". Is it copied from the cluster config to the operator config? Is the rest of the config still validated?

Reply:
Good point; clarified.